libnavigation-core/src/main/java/com/mapbox/navigation/core/routeoptions/RouteOptionsUpdater.kt

```diff
@@ -100,32 +100,15 @@ class RouteOptionsUpdater {
                                 it.addAll(approachesList.subList(index, coordinatesList.size))
                             }
                         }
-                    ).apply {
-                        if (routeOptions.profile() == DirectionsCriteria.PROFILE_DRIVING_TRAFFIC) {
```

Comment:
Was this removed on purpose? Look at https://github.com/mapbox/mapbox-navigation-android/pull/6147

cc @VysotskiVadim

Reply:
Rebase issues. Moved it back.